Dockerfile

```diff
@@ -1,18 +1,20 @@
-FROM harbor.intgdc.com/tools/gdc-java-8-jre:8291370
+FROM harbor.intgdc.com/tools/gdc-java-8-jre-centos8:6c54afc
 
 ARG RVM_VERSION=stable
 ARG JRUBY_VERSION=9.2.5.0
 
 LABEL image_name="GDC LCM Bricks"
 LABEL maintainer="LCM <lcm@gooddata.com>"
 LABEL git_repository_url="https://github.com/gooddata/gooddata-ruby/"
-LABEL parent_image="harbor.intgdc.com/tools/gdc-java-8-jre:8291370"
+LABEL parent_image="harbor.intgdc.com/tools/gdc-java-8-jre-centos8:6c54afc"
 
 # which is required by RVM
 RUN yum install -y curl which patch make git maven \
     && yum clean all \
     && rm -rf /var/cache/yum
 
+RUN yum install -y procps
```

Comment:
Should we add to "RUN yum install -y curl which patch make git maven" together?

Reply:
Done